OMCompiler/Compiler/BackEnd/RemoveSimpleEquations.mo

```diff
@@ -3137,12 +3137,12 @@ algorithm
     then BackendVariable.setVarStartValueOption(v, start);
 
     case (_, true, (_, values), BackendDAE.SHARED(globalKnownVars=globalKnownVars)) algorithm
-      v := BackendVariable.setVarFixed(inVar, true);
```

Comment:
That needs to happen independent of the status of the dumping flag, why did you move it inside?

Reply:
ah, i didn't get that. i'll move it back-